client/src/app/services/offline.service.ts

```diff
@@ -0,0 +1,26 @@
+import { Injectable } from '@angular/core';
+import { ToastController } from '@ionic/angular';
+import { TranslateService } from '@ngx-translate/core';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class OfflineService {
+  constructor(
+    public toastController: ToastController,
+    private translate: TranslateService,
+  ) {}
+
+  async presentToast() {
+    const toast = await this.toastController.create({
+      position: 'top',
+      cssClass: 'offline-toaster',
```

Comment:
```suggestion
      cssClass: 'notification',
```

"Toaster" isn't really a 'thing', I think. We should name it after its function/form, not after the tool we use for it.